cpp/log/cert.cc

```diff
@@ -243,24 +243,15 @@ unique_ptr<Cert> Cert::FromDerBio(BIO* bio_in) {
 
 
 string Cert::PrintVersion() const {
-  if (!IsLoaded()) {
-    LOG(ERROR) << "Cert not loaded";
-    return string();
-  }
-
-  const long version(X509_get_version(x509_.get()));
+  const long version(X509_get_version(CHECK_NOTNULL(x509_.get())));
```

Comment:
Hopefully, the commit before that one should make it impossible for `x509_` to ever be null, as well, so this is sheer paranoia, really.